runtime/doc/eval.txt

```diff
@@ -7008,6 +7009,28 @@ sqrt({expr})						*sqrt()*
 		"nan" may be different, it depends on system libraries.
 
 
+stdpath({what})					*stdpath()*
+		Returns the standard path(s) for {what}.
+
+		These directories are the default locations for various files
+		used by Neovim.
+
+		{what} can be one of the following:
+			'config'	User configuration directory. The
```

Comment:
Not seen in GH interface, but is not `'config'` wrongly highlighted as an option? At least this should be `""`, but better a table without any quotes.